# HG changeset patch
# Parent 49af0f9166a692dc01f580237c982def613a8bbc
-Cleanup: remove now unused STR_ABOUT_MENU_SEPARATOR string from all languages.

diff --git a/src/lang/afrikaans.txt b/src/lang/afrikaans.txt
--- a/src/lang/afrikaans.txt
+++ b/src/lang/afrikaans.txt
@@ -443,7 +443,6 @@ STR_NEWS_MENU_MESSAGE_HISTORY_MENU      
 
 ############ range for about menu starts
 STR_ABOUT_MENU_LAND_BLOCK_INFO                                  :Gebied informasie
-STR_ABOUT_MENU_SEPARATOR                                        :
 STR_ABOUT_MENU_TOGGLE_CONSOLE                                   :Skakel terminaal
 STR_ABOUT_MENU_AI_DEBUG                                         :AI/Spel skript ontfout
 STR_ABOUT_MENU_SCREENSHOT                                       :Skermskoot
diff --git a/src/lang/arabic_egypt.txt b/src/lang/arabic_egypt.txt
--- a/src/lang/arabic_egypt.txt
+++ b/src/lang/arabic_egypt.txt
@@ -439,7 +439,6 @@ STR_NEWS_MENU_MESSAGE_HISTORY_MENU      
 
 ############ range for about menu starts
 STR_ABOUT_MENU_LAND_BLOCK_INFO                                  :معلومات مربع في الخريطة
-STR_ABOUT_MENU_SEPARATOR                                        :
 STR_ABOUT_MENU_TOGGLE_CONSOLE                                   :توقل كونسول
 STR_ABOUT_MENU_AI_DEBUG                                         :مكتشف اخطاء الذكاء الصناعي
 STR_ABOUT_MENU_SCREENSHOT                                       :صورة من الشاشة - Ctrl-S -
diff --git a/src/lang/basque.txt b/src/lang/basque.txt
--- a/src/lang/basque.txt
+++ b/src/lang/basque.txt
@@ -442,7 +442,6 @@ STR_NEWS_MENU_MESSAGE_HISTORY_MENU      
 
 ############ range for about menu starts
 STR_ABOUT_MENU_LAND_BLOCK_INFO                                  :Lur arearen informazioa
-STR_ABOUT_MENU_SEPARATOR                                        :
 STR_ABOUT_MENU_TOGGLE_CONSOLE                                   :Konsola aktibatu
 STR_ABOUT_MENU_AI_DEBUG                                         :IA/Joko script-aren garbitzailea
 STR_ABOUT_MENU_SCREENSHOT                                       :Pantailan dagoenaren argazkia hartu (Ctrl+S)
diff --git a/src/lang/belarusian.txt b/src/lang/belarusian.txt
--- a/src/lang/belarusian.txt
+++ b/src/lang/belarusian.txt
@@ -756,7 +756,6 @@ STR_NEWS_MENU_MESSAGE_HISTORY_MENU      
 
 ############ range for about menu starts
 STR_ABOUT_MENU_LAND_BLOCK_INFO                                  :Зьвесткі аб зямлі
-STR_ABOUT_MENU_SEPARATOR                                        :
 STR_ABOUT_MENU_TOGGLE_CONSOLE                                   :Кансоль
 STR_ABOUT_MENU_AI_DEBUG                                         :Наладка штучнага інтэлекту (ШІ/AI) / скрыптоў
 STR_ABOUT_MENU_SCREENSHOT                                       :Здымак экрана (Ctrl+S)
diff --git a/src/lang/brazilian_portuguese.txt b/src/lang/brazilian_portuguese.txt
--- a/src/lang/brazilian_portuguese.txt
+++ b/src/lang/brazilian_portuguese.txt
@@ -458,7 +458,6 @@ STR_NEWS_MENU_MESSAGE_HISTORY_MENU      
 
 ############ range for about menu starts
 STR_ABOUT_MENU_LAND_BLOCK_INFO                                  :Informação da área do terreno
-STR_ABOUT_MENU_SEPARATOR                                        :
 STR_ABOUT_MENU_TOGGLE_CONSOLE                                   :Alternar console
 STR_ABOUT_MENU_AI_DEBUG                                         :Depurar IA/Script do jogo
 STR_ABOUT_MENU_SCREENSHOT                                       :Captura de tela
diff --git a/src/lang/bulgarian.txt b/src/lang/bulgarian.txt
--- a/src/lang/bulgarian.txt
+++ b/src/lang/bulgarian.txt
@@ -444,7 +444,6 @@ STR_NEWS_MENU_MESSAGE_HISTORY_MENU      
 
 ############ range for about menu starts
 STR_ABOUT_MENU_LAND_BLOCK_INFO                                  :Информация за терена
-STR_ABOUT_MENU_SEPARATOR                                        :
 STR_ABOUT_MENU_TOGGLE_CONSOLE                                   :Показване/скриване на конзола
 STR_ABOUT_MENU_AI_DEBUG                                         :ИИ дебъг
 STR_ABOUT_MENU_SCREENSHOT                                       :Screenshot
diff --git a/src/lang/catalan.txt b/src/lang/catalan.txt
--- a/src/lang/catalan.txt
+++ b/src/lang/catalan.txt
@@ -458,7 +458,6 @@ STR_NEWS_MENU_MESSAGE_HISTORY_MENU      
 
 ############ range for about menu starts
 STR_ABOUT_MENU_LAND_BLOCK_INFO                                  :Informació de l'àrea de terreny
-STR_ABOUT_MENU_SEPARATOR                                        :
 STR_ABOUT_MENU_TOGGLE_CONSOLE                                   :Canvia al terminal
 STR_ABOUT_MENU_AI_DEBUG                                         :Depuració de la IA/Scripts del joc
 STR_ABOUT_MENU_SCREENSHOT                                       :Captura de pantalla
diff --git a/src/lang/croatian.txt b/src/lang/croatian.txt
--- a/src/lang/croatian.txt
+++ b/src/lang/croatian.txt
@@ -553,7 +553,6 @@ STR_NEWS_MENU_MESSAGE_HISTORY_MENU      
 
 ############ range for about menu starts
 STR_ABOUT_MENU_LAND_BLOCK_INFO                                  :Podaci o zemljištu
-STR_ABOUT_MENU_SEPARATOR                                        :
 STR_ABOUT_MENU_TOGGLE_CONSOLE                                   :Otvori konzolu
 STR_ABOUT_MENU_AI_DEBUG                                         :Debugiranje UI-ja/Skripte igre
 STR_ABOUT_MENU_SCREENSHOT                                       :Slika zaslona
diff --git a/src/lang/czech.txt b/src/lang/czech.txt
--- a/src/lang/czech.txt
+++ b/src/lang/czech.txt
@@ -527,7 +527,6 @@ STR_NEWS_MENU_MESSAGE_HISTORY_MENU      
 
 ############ range for about menu starts
 STR_ABOUT_MENU_LAND_BLOCK_INFO                                  :Informace o zemi
-STR_ABOUT_MENU_SEPARATOR                                        :
 STR_ABOUT_MENU_TOGGLE_CONSOLE                                   :Zobrazit nebo skrýt konzoli
 STR_ABOUT_MENU_AI_DEBUG                                         :Ladění AI / herních skriptů
 STR_ABOUT_MENU_SCREENSHOT                                       :Screenshot
diff --git a/src/lang/danish.txt b/src/lang/danish.txt
--- a/src/lang/danish.txt
+++ b/src/lang/danish.txt
@@ -443,7 +443,6 @@ STR_NEWS_MENU_MESSAGE_HISTORY_MENU      
 
 ############ range for about menu starts
 STR_ABOUT_MENU_LAND_BLOCK_INFO                                  :Landområde information
-STR_ABOUT_MENU_SEPARATOR                                        :
 STR_ABOUT_MENU_TOGGLE_CONSOLE                                   :Tænd/Sluk konsol
 STR_ABOUT_MENU_AI_DEBUG                                         :Computerspiller/spilscript debug
 STR_ABOUT_MENU_SCREENSHOT                                       :Skærmbillede (Ctrl-S)
diff --git a/src/lang/dutch.txt b/src/lang/dutch.txt
--- a/src/lang/dutch.txt
+++ b/src/lang/dutch.txt
@@ -457,7 +457,6 @@ STR_NEWS_MENU_MESSAGE_HISTORY_MENU      
 
 ############ range for about menu starts
 STR_ABOUT_MENU_LAND_BLOCK_INFO                                  :Landinformatie
-STR_ABOUT_MENU_SEPARATOR                                        :
 STR_ABOUT_MENU_TOGGLE_CONSOLE                                   :Schakel console aan/uit
 STR_ABOUT_MENU_AI_DEBUG                                         :AI/Game Script debug
 STR_ABOUT_MENU_SCREENSHOT                                       :Screenshot
diff --git a/src/lang/english_AU.txt b/src/lang/english_AU.txt
--- a/src/lang/english_AU.txt
+++ b/src/lang/english_AU.txt
@@ -457,7 +457,6 @@ STR_NEWS_MENU_MESSAGE_HISTORY_MENU      
 
 ############ range for about menu starts
 STR_ABOUT_MENU_LAND_BLOCK_INFO                                  :Land area information
-STR_ABOUT_MENU_SEPARATOR                                        :
 STR_ABOUT_MENU_TOGGLE_CONSOLE                                   :Toggle console
 STR_ABOUT_MENU_AI_DEBUG                                         :AI/Game script debug
 STR_ABOUT_MENU_SCREENSHOT                                       :Screenshot
diff --git a/src/lang/english_US.txt b/src/lang/english_US.txt
--- a/src/lang/english_US.txt
+++ b/src/lang/english_US.txt
@@ -457,7 +457,6 @@ STR_NEWS_MENU_MESSAGE_HISTORY_MENU      
 
 ############ range for about menu starts
 STR_ABOUT_MENU_LAND_BLOCK_INFO                                  :Land area information
-STR_ABOUT_MENU_SEPARATOR                                        :
 STR_ABOUT_MENU_TOGGLE_CONSOLE                                   :Toggle console
 STR_ABOUT_MENU_AI_DEBUG                                         :AI/Game script debug
 STR_ABOUT_MENU_SCREENSHOT                                       :Screenshot
diff --git a/src/lang/esperanto.txt b/src/lang/esperanto.txt
--- a/src/lang/esperanto.txt
+++ b/src/lang/esperanto.txt
@@ -440,7 +440,6 @@ STR_NEWS_MENU_MESSAGE_HISTORY_MENU      
 
 ############ range for about menu starts
 STR_ABOUT_MENU_LAND_BLOCK_INFO                                  :Landkvadrataj informoj
-STR_ABOUT_MENU_SEPARATOR                                        :
 STR_ABOUT_MENU_TOGGLE_CONSOLE                                   :Baskuligi Konzolon
 STR_ABOUT_MENU_AI_DEBUG                                         :AI/Ludo skripto sencimigo
 STR_ABOUT_MENU_SCREENSHOT                                       :Ekranfoto
diff --git a/src/lang/estonian.txt b/src/lang/estonian.txt
--- a/src/lang/estonian.txt
+++ b/src/lang/estonian.txt
@@ -514,7 +514,6 @@ STR_NEWS_MENU_MESSAGE_HISTORY_MENU      
 
 ############ range for about menu starts
 STR_ABOUT_MENU_LAND_BLOCK_INFO                                  :Maa-ala andmed
-STR_ABOUT_MENU_SEPARATOR                                        :
 STR_ABOUT_MENU_TOGGLE_CONSOLE                                   :Lülita konsool sisse/välja
 STR_ABOUT_MENU_AI_DEBUG                                         :Arvutivea otsing
 STR_ABOUT_MENU_SCREENSHOT                                       :Ekraanitõmmis
diff --git a/src/lang/faroese.txt b/src/lang/faroese.txt
--- a/src/lang/faroese.txt
+++ b/src/lang/faroese.txt
@@ -444,7 +444,6 @@ STR_NEWS_MENU_MESSAGE_HISTORY_MENU      
 
 ############ range for about menu starts
 STR_ABOUT_MENU_LAND_BLOCK_INFO                                  :Landa økis kunning
-STR_ABOUT_MENU_SEPARATOR                                        :
 STR_ABOUT_MENU_TOGGLE_CONSOLE                                   :Console opið ella lukka
 STR_ABOUT_MENU_AI_DEBUG                                         :AI/Spæl script debug
 STR_ABOUT_MENU_SCREENSHOT                                       :Skermmynd
diff --git a/src/lang/finnish.txt b/src/lang/finnish.txt
--- a/src/lang/finnish.txt
+++ b/src/lang/finnish.txt
@@ -457,7 +457,6 @@ STR_NEWS_MENU_MESSAGE_HISTORY_MENU      
 
 ############ range for about menu starts
 STR_ABOUT_MENU_LAND_BLOCK_INFO                                  :Maa-alueen tiedot
-STR_ABOUT_MENU_SEPARATOR                                        :
 STR_ABOUT_MENU_TOGGLE_CONSOLE                                   :Konsoli
 STR_ABOUT_MENU_AI_DEBUG                                         :Tekoälyn/Peliskriptin virheenjäljitys
 STR_ABOUT_MENU_SCREENSHOT                                       :Kuvakaappaus
diff --git a/src/lang/french.txt b/src/lang/french.txt
--- a/src/lang/french.txt
+++ b/src/lang/french.txt
@@ -458,7 +458,6 @@ STR_NEWS_MENU_MESSAGE_HISTORY_MENU      
 
 ############ range for about menu starts
 STR_ABOUT_MENU_LAND_BLOCK_INFO                                  :Outil d'inspection du paysage
-STR_ABOUT_MENU_SEPARATOR                                        :
 STR_ABOUT_MENU_TOGGLE_CONSOLE                                   :Afficher/Cacher la console
 STR_ABOUT_MENU_AI_DEBUG                                         :Débogage de scripts
 STR_ABOUT_MENU_SCREENSHOT                                       :Copie d'écran
diff --git a/src/lang/gaelic.txt b/src/lang/gaelic.txt
--- a/src/lang/gaelic.txt
+++ b/src/lang/gaelic.txt
@@ -645,7 +645,6 @@ STR_NEWS_MENU_MESSAGE_HISTORY_MENU      
 
 ############ range for about menu starts
 STR_ABOUT_MENU_LAND_BLOCK_INFO                                  :Fiosrachadh raon tìre
-STR_ABOUT_MENU_SEPARATOR                                        :
 STR_ABOUT_MENU_TOGGLE_CONSOLE                                   :Toglaich a' chonsoil
 STR_ABOUT_MENU_AI_DEBUG                                         :Dì-bhugaich IF/sgriobt geama
 STR_ABOUT_MENU_SCREENSHOT                                       :Glacadh-sgrìn
diff --git a/src/lang/galician.txt b/src/lang/galician.txt
--- a/src/lang/galician.txt
+++ b/src/lang/galician.txt
@@ -458,7 +458,6 @@ STR_NEWS_MENU_MESSAGE_HISTORY_MENU      
 
 ############ range for about menu starts
 STR_ABOUT_MENU_LAND_BLOCK_INFO                                  :Información de área de terreo
-STR_ABOUT_MENU_SEPARATOR                                        :
 STR_ABOUT_MENU_TOGGLE_CONSOLE                                   :(Des)Activar consola
 STR_ABOUT_MENU_AI_DEBUG                                         :Depuración IA/script do xogo
 STR_ABOUT_MENU_SCREENSHOT                                       :Captura de pantalla
diff --git a/src/lang/german.txt b/src/lang/german.txt
--- a/src/lang/german.txt
+++ b/src/lang/german.txt
@@ -458,7 +458,6 @@ STR_NEWS_MENU_MESSAGE_HISTORY_MENU      
 
 ############ range for about menu starts
 STR_ABOUT_MENU_LAND_BLOCK_INFO                                  :Gebietsinformationen
-STR_ABOUT_MENU_SEPARATOR                                        :
 STR_ABOUT_MENU_TOGGLE_CONSOLE                                   :Konsole öffnen/schließen
 STR_ABOUT_MENU_AI_DEBUG                                         :KI / Skript-Debug
 STR_ABOUT_MENU_SCREENSHOT                                       :Screenshot (Standard: Strg+S)
diff --git a/src/lang/greek.txt b/src/lang/greek.txt
--- a/src/lang/greek.txt
+++ b/src/lang/greek.txt
@@ -506,7 +506,6 @@ STR_NEWS_MENU_MESSAGE_HISTORY_MENU      
 
 ############ range for about menu starts
 STR_ABOUT_MENU_LAND_BLOCK_INFO                                  :Πληροφορίες περιοχής εδάφους
-STR_ABOUT_MENU_SEPARATOR                                        :
 STR_ABOUT_MENU_TOGGLE_CONSOLE                                   :Εναλλαγή κονσόλας
 STR_ABOUT_MENU_AI_DEBUG                                         :Αποσφαλμάτωση AI και δέσμης ενεργειών παιχνιδιού
 STR_ABOUT_MENU_SCREENSHOT                                       :Στιγμιότυπο οθόνης
diff --git a/src/lang/hebrew.txt b/src/lang/hebrew.txt
--- a/src/lang/hebrew.txt
+++ b/src/lang/hebrew.txt
@@ -456,7 +456,6 @@ STR_NEWS_MENU_MESSAGE_HISTORY_MENU      
 
 ############ range for about menu starts
 STR_ABOUT_MENU_LAND_BLOCK_INFO                                  :מידע על שטח קרקע
-STR_ABOUT_MENU_SEPARATOR                                        :
 STR_ABOUT_MENU_TOGGLE_CONSOLE                                   :פתח/סגור קונסולה
 STR_ABOUT_MENU_AI_DEBUG                                         :דיבאג של סקריפטים של בינה מלאכותית או של המשחק
 STR_ABOUT_MENU_SCREENSHOT                                       :צילום מסך
diff --git a/src/lang/hungarian.txt b/src/lang/hungarian.txt
--- a/src/lang/hungarian.txt
+++ b/src/lang/hungarian.txt
@@ -520,7 +520,6 @@ STR_NEWS_MENU_MESSAGE_HISTORY_MENU      
 
 ############ range for about menu starts
 STR_ABOUT_MENU_LAND_BLOCK_INFO                                  :Terület-információ
-STR_ABOUT_MENU_SEPARATOR                                        :
 STR_ABOUT_MENU_TOGGLE_CONSOLE                                   :Átváltás konzolra
 STR_ABOUT_MENU_AI_DEBUG                                         :MI / Játékszkript nyomkövetés
 STR_ABOUT_MENU_SCREENSHOT                                       :Képmentés
diff --git a/src/lang/icelandic.txt b/src/lang/icelandic.txt
--- a/src/lang/icelandic.txt
+++ b/src/lang/icelandic.txt
@@ -444,7 +444,6 @@ STR_NEWS_MENU_MESSAGE_HISTORY_MENU      
 
 ############ range for about menu starts
 STR_ABOUT_MENU_LAND_BLOCK_INFO                                  :Landssvæðisupplýsingar
-STR_ABOUT_MENU_SEPARATOR                                        :
 STR_ABOUT_MENU_TOGGLE_CONSOLE                                   :Kveikja á stýriskjá
 STR_ABOUT_MENU_AI_DEBUG                                         :Aflúsun gervigreindar/forskrifta
 STR_ABOUT_MENU_SCREENSHOT                                       :Skjámynd
diff --git a/src/lang/indonesian.txt b/src/lang/indonesian.txt
--- a/src/lang/indonesian.txt
+++ b/src/lang/indonesian.txt
@@ -443,7 +443,6 @@ STR_NEWS_MENU_MESSAGE_HISTORY_MENU      
 
 ############ range for about menu starts
 STR_ABOUT_MENU_LAND_BLOCK_INFO                                  :Informasi area daratan
-STR_ABOUT_MENU_SEPARATOR                                        :
 STR_ABOUT_MENU_TOGGLE_CONSOLE                                   :Hidup/matikan Layar Konsol
 STR_ABOUT_MENU_AI_DEBUG                                         :Debug skrip AI
 STR_ABOUT_MENU_SCREENSHOT                                       :Ambil gambar
diff --git a/src/lang/irish.txt b/src/lang/irish.txt
--- a/src/lang/irish.txt
+++ b/src/lang/irish.txt
@@ -439,7 +439,6 @@ STR_NEWS_MENU_MESSAGE_HISTORY_MENU      
 
 ############ range for about menu starts
 STR_ABOUT_MENU_LAND_BLOCK_INFO                                  :Faisnéis faoin limistéar talún
-STR_ABOUT_MENU_SEPARATOR                                        :
 STR_ABOUT_MENU_TOGGLE_CONSOLE                                   :Scoránaigh consól
 STR_ABOUT_MENU_AI_DEBUG                                         :Dífhabhtú scripteanna AI/Cluiche
 STR_ABOUT_MENU_SCREENSHOT                                       :Seat scáileáin
diff --git a/src/lang/italian.txt b/src/lang/italian.txt
--- a/src/lang/italian.txt
+++ b/src/lang/italian.txt
@@ -459,7 +459,6 @@ STR_NEWS_MENU_MESSAGE_HISTORY_MENU      
 
 ############ range for about menu starts
 STR_ABOUT_MENU_LAND_BLOCK_INFO                                  :Informazioni sull'area di terreno
-STR_ABOUT_MENU_SEPARATOR                                        :
 STR_ABOUT_MENU_TOGGLE_CONSOLE                                   :Apri/chiudi console
 STR_ABOUT_MENU_AI_DEBUG                                         :Debug IA/Script
 STR_ABOUT_MENU_SCREENSHOT                                       :Screenshot
diff --git a/src/lang/japanese.txt b/src/lang/japanese.txt
--- a/src/lang/japanese.txt
+++ b/src/lang/japanese.txt
@@ -457,7 +457,6 @@ STR_NEWS_MENU_MESSAGE_HISTORY_MENU      
 
 ############ range for about menu starts
 STR_ABOUT_MENU_LAND_BLOCK_INFO                                  :地域情報
-STR_ABOUT_MENU_SEPARATOR                                        :―――――――――――――――――
 STR_ABOUT_MENU_TOGGLE_CONSOLE                                   :コンソールの表示切替
 STR_ABOUT_MENU_AI_DEBUG                                         :AI/ゲームスクリプトのデバッグ
 STR_ABOUT_MENU_SCREENSHOT                                       :スクリーンショット撮影(現在のズーム)
diff --git a/src/lang/korean.txt b/src/lang/korean.txt
--- a/src/lang/korean.txt
+++ b/src/lang/korean.txt
@@ -458,7 +458,6 @@ STR_NEWS_MENU_MESSAGE_HISTORY_MENU      
 
 ############ range for about menu starts
 STR_ABOUT_MENU_LAND_BLOCK_INFO                                  :지형 정보
-STR_ABOUT_MENU_SEPARATOR                                        :
 STR_ABOUT_MENU_TOGGLE_CONSOLE                                   :콘솔 켜기/끄기
 STR_ABOUT_MENU_AI_DEBUG                                         :인공지능/게임 스크립트 디버그
 STR_ABOUT_MENU_SCREENSHOT                                       :스크린샷 찍기
diff --git a/src/lang/latvian.txt b/src/lang/latvian.txt
--- a/src/lang/latvian.txt
+++ b/src/lang/latvian.txt
@@ -445,7 +445,6 @@ STR_NEWS_MENU_MESSAGE_HISTORY_MENU      
 
 ############ range for about menu starts
 STR_ABOUT_MENU_LAND_BLOCK_INFO                                  :Zemes platības informācija
-STR_ABOUT_MENU_SEPARATOR                                        :
 STR_ABOUT_MENU_TOGGLE_CONSOLE                                   :Atvērt/aizvērt konsoli
 STR_ABOUT_MENU_AI_DEBUG                                         :MI/spēles skriptu atkļūdošana
 STR_ABOUT_MENU_SCREENSHOT                                       :Ekrānuzņēmums
diff --git a/src/lang/lithuanian.txt b/src/lang/lithuanian.txt
--- a/src/lang/lithuanian.txt
+++ b/src/lang/lithuanian.txt
@@ -641,7 +641,6 @@ STR_NEWS_MENU_MESSAGE_HISTORY_MENU      
 
 ############ range for about menu starts
 STR_ABOUT_MENU_LAND_BLOCK_INFO                                  :Žemės ploto informacija
-STR_ABOUT_MENU_SEPARATOR                                        :
 STR_ABOUT_MENU_TOGGLE_CONSOLE                                   :Perjungti konsolę
 STR_ABOUT_MENU_AI_DEBUG                                         :AI / GameScript derinimas
 STR_ABOUT_MENU_SCREENSHOT                                       :Ekrano nuotrauka
diff --git a/src/lang/luxembourgish.txt b/src/lang/luxembourgish.txt
--- a/src/lang/luxembourgish.txt
+++ b/src/lang/luxembourgish.txt
@@ -448,7 +448,6 @@ STR_NEWS_MENU_MESSAGE_HISTORY_MENU      
 
 ############ range for about menu starts
 STR_ABOUT_MENU_LAND_BLOCK_INFO                                  :Landinformatiounen
-STR_ABOUT_MENU_SEPARATOR                                        :
 STR_ABOUT_MENU_TOGGLE_CONSOLE                                   :Konsole un/aus
 STR_ABOUT_MENU_AI_DEBUG                                         :KI / Spill-Script Debug
 STR_ABOUT_MENU_SCREENSHOT                                       :Screenshot (Ctrl+S)
diff --git a/src/lang/malay.txt b/src/lang/malay.txt
--- a/src/lang/malay.txt
+++ b/src/lang/malay.txt
@@ -442,7 +442,6 @@ STR_NEWS_MENU_MESSAGE_HISTORY_MENU      
 
 ############ range for about menu starts
 STR_ABOUT_MENU_LAND_BLOCK_INFO                                  :Maklumat kawasan tanah
-STR_ABOUT_MENU_SEPARATOR                                        :
 STR_ABOUT_MENU_TOGGLE_CONSOLE                                   :Papar atau padamkan konsol
 STR_ABOUT_MENU_AI_DEBUG                                         :Al/Skrip pepijat permainan
 STR_ABOUT_MENU_SCREENSHOT                                       :Tangkapan skrin (Ctrl+S)
diff --git a/src/lang/norwegian_bokmal.txt b/src/lang/norwegian_bokmal.txt
--- a/src/lang/norwegian_bokmal.txt
+++ b/src/lang/norwegian_bokmal.txt
@@ -459,7 +459,6 @@ STR_NEWS_MENU_MESSAGE_HISTORY_MENU      
 
 ############ range for about menu starts
 STR_ABOUT_MENU_LAND_BLOCK_INFO                                  :Informasjon om landområde
-STR_ABOUT_MENU_SEPARATOR                                        :
 STR_ABOUT_MENU_TOGGLE_CONSOLE                                   :Vis/skjul konsoll
 STR_ABOUT_MENU_AI_DEBUG                                         :AI/Spillskript-feilsøking
 STR_ABOUT_MENU_SCREENSHOT                                       :Skjermbilde
diff --git a/src/lang/norwegian_nynorsk.txt b/src/lang/norwegian_nynorsk.txt
--- a/src/lang/norwegian_nynorsk.txt
+++ b/src/lang/norwegian_nynorsk.txt
@@ -444,7 +444,6 @@ STR_NEWS_MENU_MESSAGE_HISTORY_MENU      
 
 ############ range for about menu starts
 STR_ABOUT_MENU_LAND_BLOCK_INFO                                  :Informasjon om landområde
-STR_ABOUT_MENU_SEPARATOR                                        :
 STR_ABOUT_MENU_TOGGLE_CONSOLE                                   :Syne/gøym konsoll
 STR_ABOUT_MENU_AI_DEBUG                                         :AI- / Spelscriptfeilsøking
 STR_ABOUT_MENU_SCREENSHOT                                       :Skjermdump
diff --git a/src/lang/polish.txt b/src/lang/polish.txt
--- a/src/lang/polish.txt
+++ b/src/lang/polish.txt
@@ -834,7 +834,6 @@ STR_NEWS_MENU_MESSAGE_HISTORY_MENU      
 
 ############ range for about menu starts
 STR_ABOUT_MENU_LAND_BLOCK_INFO                                  :Informacje o terenie
-STR_ABOUT_MENU_SEPARATOR                                        :
 STR_ABOUT_MENU_TOGGLE_CONSOLE                                   :Przełącz konsolę
 STR_ABOUT_MENU_AI_DEBUG                                         :Debugowanie SI / Game Script
 STR_ABOUT_MENU_SCREENSHOT                                       :Zrzut ekranu
diff --git a/src/lang/portuguese.txt b/src/lang/portuguese.txt
--- a/src/lang/portuguese.txt
+++ b/src/lang/portuguese.txt
@@ -452,7 +452,6 @@ STR_NEWS_MENU_MESSAGE_HISTORY_MENU      
 
 ############ range for about menu starts
 STR_ABOUT_MENU_LAND_BLOCK_INFO                                  :Informações do terreno
-STR_ABOUT_MENU_SEPARATOR                                        :
 STR_ABOUT_MENU_TOGGLE_CONSOLE                                   :Mostrar/ocultar consola
 STR_ABOUT_MENU_AI_DEBUG                                         :Depuração da IA/Scripts de Jogo
 STR_ABOUT_MENU_SCREENSHOT                                       :Captura de ecrã
diff --git a/src/lang/romanian.txt b/src/lang/romanian.txt
--- a/src/lang/romanian.txt
+++ b/src/lang/romanian.txt
@@ -444,7 +444,6 @@ STR_NEWS_MENU_MESSAGE_HISTORY_MENU      
 
 ############ range for about menu starts
 STR_ABOUT_MENU_LAND_BLOCK_INFO                                  :Informaţii despre teren
-STR_ABOUT_MENU_SEPARATOR                                        :
 STR_ABOUT_MENU_TOGGLE_CONSOLE                                   :Consolă pornit/oprit
 STR_ABOUT_MENU_AI_DEBUG                                         :Depanare Inteligenţă Artificială / Script Joc
 STR_ABOUT_MENU_SCREENSHOT                                       :Capturează ecranul
diff --git a/src/lang/russian.txt b/src/lang/russian.txt
--- a/src/lang/russian.txt
+++ b/src/lang/russian.txt
@@ -583,7 +583,6 @@ STR_NEWS_MENU_MESSAGE_HISTORY_MENU      
 
 ############ range for about menu starts
 STR_ABOUT_MENU_LAND_BLOCK_INFO                                  :Информация о земле
-STR_ABOUT_MENU_SEPARATOR                                        :
 STR_ABOUT_MENU_TOGGLE_CONSOLE                                   :Консоль
 STR_ABOUT_MENU_AI_DEBUG                                         :Отладка ИИ / скриптов
 STR_ABOUT_MENU_SCREENSHOT                                       :Снимок экрана
diff --git a/src/lang/serbian.txt b/src/lang/serbian.txt
--- a/src/lang/serbian.txt
+++ b/src/lang/serbian.txt
@@ -638,7 +638,6 @@ STR_NEWS_MENU_MESSAGE_HISTORY_MENU      
 
 ############ range for about menu starts
 STR_ABOUT_MENU_LAND_BLOCK_INFO                                  :Podaci o zemljištu
-STR_ABOUT_MENU_SEPARATOR                                        :
 STR_ABOUT_MENU_TOGGLE_CONSOLE                                   :Uključi/isključi konzolu
 STR_ABOUT_MENU_AI_DEBUG                                         :Korekcija veštačke inteligencije / skripte partije
 STR_ABOUT_MENU_SCREENSHOT                                       :Sačuvaj sliku
diff --git a/src/lang/simplified_chinese.txt b/src/lang/simplified_chinese.txt
--- a/src/lang/simplified_chinese.txt
+++ b/src/lang/simplified_chinese.txt
@@ -457,7 +457,6 @@ STR_NEWS_MENU_MESSAGE_HISTORY_MENU      
 
 ############ range for about menu starts
 STR_ABOUT_MENU_LAND_BLOCK_INFO                                  :查询地块信息
-STR_ABOUT_MENU_SEPARATOR                                        :
 STR_ABOUT_MENU_TOGGLE_CONSOLE                                   :切换到控制台模式
 STR_ABOUT_MENU_AI_DEBUG                                         :AI /游戏脚本调试
 STR_ABOUT_MENU_SCREENSHOT                                       :屏幕截图
diff --git a/src/lang/slovak.txt b/src/lang/slovak.txt
--- a/src/lang/slovak.txt
+++ b/src/lang/slovak.txt
@@ -521,7 +521,6 @@ STR_NEWS_MENU_MESSAGE_HISTORY_MENU      
 
 ############ range for about menu starts
 STR_ABOUT_MENU_LAND_BLOCK_INFO                                  :Informácie o pozemku
-STR_ABOUT_MENU_SEPARATOR                                        :
 STR_ABOUT_MENU_TOGGLE_CONSOLE                                   :Zobraziť / skryť konzolu
 STR_ABOUT_MENU_AI_DEBUG                                         :Umelá inteligencia / Ladenie skriptov
 STR_ABOUT_MENU_SCREENSHOT                                       :Snímka
diff --git a/src/lang/slovenian.txt b/src/lang/slovenian.txt
--- a/src/lang/slovenian.txt
+++ b/src/lang/slovenian.txt
@@ -595,7 +595,6 @@ STR_NEWS_MENU_MESSAGE_HISTORY_MENU      
 
 ############ range for about menu starts
 STR_ABOUT_MENU_LAND_BLOCK_INFO                                  :Informacije o terenu
-STR_ABOUT_MENU_SEPARATOR                                        :
 STR_ABOUT_MENU_TOGGLE_CONSOLE                                   :Vklopi/Izklopi konzolo
 STR_ABOUT_MENU_AI_DEBUG                                         :Razhročevanje UI / skript
 STR_ABOUT_MENU_SCREENSHOT                                       :Zajemi sliko
diff --git a/src/lang/spanish.txt b/src/lang/spanish.txt
--- a/src/lang/spanish.txt
+++ b/src/lang/spanish.txt
@@ -458,7 +458,6 @@ STR_NEWS_MENU_MESSAGE_HISTORY_MENU      
 
 ############ range for about menu starts
 STR_ABOUT_MENU_LAND_BLOCK_INFO                                  :Información sobre terreno
-STR_ABOUT_MENU_SEPARATOR                                        :
 STR_ABOUT_MENU_TOGGLE_CONSOLE                                   :Activar consola
 STR_ABOUT_MENU_AI_DEBUG                                         :Depuración de Script de IA / Juego
 STR_ABOUT_MENU_SCREENSHOT                                       :Captura de pantalla
diff --git a/src/lang/swedish.txt b/src/lang/swedish.txt
--- a/src/lang/swedish.txt
+++ b/src/lang/swedish.txt
@@ -448,7 +448,6 @@ STR_NEWS_MENU_MESSAGE_HISTORY_MENU      
 
 ############ range for about menu starts
 STR_ABOUT_MENU_LAND_BLOCK_INFO                                  :Information om mark
-STR_ABOUT_MENU_SEPARATOR                                        :
 STR_ABOUT_MENU_TOGGLE_CONSOLE                                   :Visa/dölj konsolen
 STR_ABOUT_MENU_AI_DEBUG                                         :Felsökning av AI / spelskript
 STR_ABOUT_MENU_SCREENSHOT                                       :Skärmdump
diff --git a/src/lang/tamil.txt b/src/lang/tamil.txt
--- a/src/lang/tamil.txt
+++ b/src/lang/tamil.txt
@@ -454,7 +454,6 @@ STR_NEWS_MENU_MESSAGE_HISTORY_MENU      
 
 ############ range for about menu starts
 STR_ABOUT_MENU_LAND_BLOCK_INFO                                  :நிலா பகுதி விவரம்
-STR_ABOUT_MENU_SEPARATOR                                        :
 STR_ABOUT_MENU_TOGGLE_CONSOLE                                   :முனையத்தை மாற்று
 STR_ABOUT_MENU_AI_DEBUG                                         :AI/வரிவடிவம் சரிபார்
 STR_ABOUT_MENU_SCREENSHOT                                       :திரைபிடிப்பு
diff --git a/src/lang/thai.txt b/src/lang/thai.txt
--- a/src/lang/thai.txt
+++ b/src/lang/thai.txt
@@ -457,7 +457,6 @@ STR_NEWS_MENU_MESSAGE_HISTORY_MENU      
 
 ############ range for about menu starts
 STR_ABOUT_MENU_LAND_BLOCK_INFO                                  :ข้อมูลพื้นที่
-STR_ABOUT_MENU_SEPARATOR                                        :
 STR_ABOUT_MENU_TOGGLE_CONSOLE                                   :เปิด/ปิด คอนโซล
 STR_ABOUT_MENU_AI_DEBUG                                         :ดีบัก สคริปต์ AI/Game
 STR_ABOUT_MENU_SCREENSHOT                                       :จับภาพหน้าจอ (Ctrl+S)
diff --git a/src/lang/traditional_chinese.txt b/src/lang/traditional_chinese.txt
--- a/src/lang/traditional_chinese.txt
+++ b/src/lang/traditional_chinese.txt
@@ -457,7 +457,6 @@ STR_NEWS_MENU_MESSAGE_HISTORY_MENU      
 
 ############ range for about menu starts
 STR_ABOUT_MENU_LAND_BLOCK_INFO                                  :土地資訊
-STR_ABOUT_MENU_SEPARATOR                                        :
 STR_ABOUT_MENU_TOGGLE_CONSOLE                                   :切換主控台
 STR_ABOUT_MENU_AI_DEBUG                                         :AI/遊戲腳本除錯
 STR_ABOUT_MENU_SCREENSHOT                                       :擷取畫面
diff --git a/src/lang/turkish.txt b/src/lang/turkish.txt
--- a/src/lang/turkish.txt
+++ b/src/lang/turkish.txt
@@ -458,7 +458,6 @@ STR_NEWS_MENU_MESSAGE_HISTORY_MENU      
 
 ############ range for about menu starts
 STR_ABOUT_MENU_LAND_BLOCK_INFO                                  :Arazi bilgisi
-STR_ABOUT_MENU_SEPARATOR                                        :
 STR_ABOUT_MENU_TOGGLE_CONSOLE                                   :Konsolu aç/kapa
 STR_ABOUT_MENU_AI_DEBUG                                         :YZ/Oyun betik hata ayıklama
 STR_ABOUT_MENU_SCREENSHOT                                       :Ekran Görüntüsü
diff --git a/src/lang/ukrainian.txt b/src/lang/ukrainian.txt
--- a/src/lang/ukrainian.txt
+++ b/src/lang/ukrainian.txt
@@ -569,7 +569,6 @@ STR_NEWS_MENU_MESSAGE_HISTORY_MENU      
 
 ############ range for about menu starts
 STR_ABOUT_MENU_LAND_BLOCK_INFO                                  :Інформація про ділянку
-STR_ABOUT_MENU_SEPARATOR                                        :
 STR_ABOUT_MENU_TOGGLE_CONSOLE                                   :Вкл./відкл. консоль
 STR_ABOUT_MENU_AI_DEBUG                                         :Налагодження АІ / Ігрового Скрипта
 STR_ABOUT_MENU_SCREENSHOT                                       :Знімок екрану
diff --git a/src/lang/unfinished/chuvash.txt b/src/lang/unfinished/chuvash.txt
--- a/src/lang/unfinished/chuvash.txt
+++ b/src/lang/unfinished/chuvash.txt
@@ -269,7 +269,6 @@ STR_TOOLBAR_SOUND_MUSIC                 
 ############ range ends here
 
 ############ range for about menu starts
-STR_ABOUT_MENU_SEPARATOR                                        :
 STR_ABOUT_MENU_SCREENSHOT                                       :Экран сӑнӗ
 STR_ABOUT_MENU_ZOOMIN_SCREENSHOT                                :Яланхилле экран сӑнӗ
 STR_ABOUT_MENU_ABOUT_OPENTTD                                    :'OpenTTD' çинчен
diff --git a/src/lang/unfinished/frisian.txt b/src/lang/unfinished/frisian.txt
--- a/src/lang/unfinished/frisian.txt
+++ b/src/lang/unfinished/frisian.txt
@@ -437,7 +437,6 @@ STR_NEWS_MENU_MESSAGE_HISTORY_MENU      
 
 ############ range for about menu starts
 STR_ABOUT_MENU_LAND_BLOCK_INFO                                  :Gebiets ynformaasje
-STR_ABOUT_MENU_SEPARATOR                                        :
 STR_ABOUT_MENU_TOGGLE_CONSOLE                                   :Skeakel console oan/ût
 STR_ABOUT_MENU_AI_DEBUG                                         :AI/Gamescript debug
 STR_ABOUT_MENU_SCREENSHOT                                       :Skermfoto
diff --git a/src/lang/unfinished/macedonian.txt b/src/lang/unfinished/macedonian.txt
--- a/src/lang/unfinished/macedonian.txt
+++ b/src/lang/unfinished/macedonian.txt
@@ -440,7 +440,6 @@ STR_NEWS_MENU_MESSAGE_HISTORY_MENU      
 
 ############ range for about menu starts
 STR_ABOUT_MENU_LAND_BLOCK_INFO                                  :Земјиште во областа
-STR_ABOUT_MENU_SEPARATOR                                        :
 STR_ABOUT_MENU_TOGGLE_CONSOLE                                   :Вклучи конзола
 STR_ABOUT_MENU_AI_DEBUG                                         :АИ / игри сценарио де-бубачки
 STR_ABOUT_MENU_SCREENSHOT                                       :Слика од екранот
diff --git a/src/lang/unfinished/marathi.txt b/src/lang/unfinished/marathi.txt
--- a/src/lang/unfinished/marathi.txt
+++ b/src/lang/unfinished/marathi.txt
@@ -363,7 +363,6 @@ STR_NEWS_MENU_MESSAGE_HISTORY_MENU      
 
 ############ range for about menu starts
 STR_ABOUT_MENU_LAND_BLOCK_INFO                                  :जागाची माहिती
-STR_ABOUT_MENU_SEPARATOR                                        :
 STR_ABOUT_MENU_ABOUT_OPENTTD                                    :ओपेन टीटीडी बद्दल
 ############ range ends here
 
diff --git a/src/lang/unfinished/persian.txt b/src/lang/unfinished/persian.txt
--- a/src/lang/unfinished/persian.txt
+++ b/src/lang/unfinished/persian.txt
@@ -447,7 +447,6 @@ STR_NEWS_MENU_MESSAGE_HISTORY_MENU      
 
 ############ range for about menu starts
 STR_ABOUT_MENU_LAND_BLOCK_INFO                                  :اطلاعات زمین ها
-STR_ABOUT_MENU_SEPARATOR                                        :
 STR_ABOUT_MENU_TOGGLE_CONSOLE                                   :باز و بسته کردن کنسول
 STR_ABOUT_MENU_AI_DEBUG                                         :خطایابی هوش مصنوعی
 STR_ABOUT_MENU_SCREENSHOT                                       :تصویر از بازی
diff --git a/src/lang/unfinished/urdu.txt b/src/lang/unfinished/urdu.txt
--- a/src/lang/unfinished/urdu.txt
+++ b/src/lang/unfinished/urdu.txt
@@ -440,7 +440,6 @@ STR_NEWS_MENU_MESSAGE_HISTORY_MENU      
 
 ############ range for about menu starts
 STR_ABOUT_MENU_LAND_BLOCK_INFO                                  :زمینی رقبہ کی معلومات
-STR_ABOUT_MENU_SEPARATOR                                        :
 STR_ABOUT_MENU_TOGGLE_CONSOLE                                   :حائطھ تدویم
 STR_ABOUT_MENU_AI_DEBUG                                         :AI debug
 STR_ABOUT_MENU_SCREENSHOT                                       :اسکرین کی تصویر
diff --git a/src/lang/vietnamese.txt b/src/lang/vietnamese.txt
--- a/src/lang/vietnamese.txt
+++ b/src/lang/vietnamese.txt
@@ -457,7 +457,6 @@ STR_NEWS_MENU_MESSAGE_HISTORY_MENU      
 
 ############ range for about menu starts
 STR_ABOUT_MENU_LAND_BLOCK_INFO                                  :Thông tin vùng đất
-STR_ABOUT_MENU_SEPARATOR                                        :
 STR_ABOUT_MENU_TOGGLE_CONSOLE                                   :Bật/tắt bảng lệnh
 STR_ABOUT_MENU_AI_DEBUG                                         :Gỡ rối AI / Game script
 STR_ABOUT_MENU_SCREENSHOT                                       :Ảnh chụp màn hình
diff --git a/src/lang/welsh.txt b/src/lang/welsh.txt
--- a/src/lang/welsh.txt
+++ b/src/lang/welsh.txt
@@ -445,7 +445,6 @@ STR_NEWS_MENU_MESSAGE_HISTORY_MENU      
 
 ############ range for about menu starts
 STR_ABOUT_MENU_LAND_BLOCK_INFO                                  :Gwybodaeth ardal tir
-STR_ABOUT_MENU_SEPARATOR                                        :
 STR_ABOUT_MENU_TOGGLE_CONSOLE                                   :Toglu Consol
 STR_ABOUT_MENU_AI_DEBUG                                         :Dadnamu AI / Sgript Gêm
 STR_ABOUT_MENU_SCREENSHOT                                       :Ciplun
